config.py.sample

```diff
@@ -0,0 +1,19 @@
+import os
+
+
+DEBUG = True
+DEFAULT_LIMIT = 10
+DEFAULT_OFFSET = 0
+FROM_EMAIL = 'YOUR@EMAIL.COM'
+MAX_EMAIL_ADDRESS_SIZE = 256
+MAX_RETRIES = 3
+MAX_SUBJECT_SIZE = 256
+MAX_TEXT_SIZE = 4096
+POSTGRESQL_HOST = 'localhost'
+RETRY_DELAY_MS = 10 * 60 * 1000  # 10 min in milliseconds
+RABBITMQ_HOST = 'localhost'
```

Comment:
The way that the queues are currently configured, yes, messages will be recovered.  The queues are declared as durable, and the messages are marked as persistent, so rabbit will save all messages to disk.  When the rabbit process is restarted, it will recover the messages from disk. 

There is a small chance that messages will be lost if for some reason the process dies in the small amount of time between receiving the message, and writing it to disk.  There are some options to work around this, but they're not configured in this app.

Rabbit does also have a clustering option, where many nodes can be configured to replicate queues among them.